### ISSUE TITLE
Unbound TI from Clearing Cells and Possible Server Crashing

### ISSUE THREAD

**Issue by RobbyMeyers** [cubewise-code/tm1py#965]
**Describe the bug**
We are using a tm1.cells.clear function and are noticing in our logs that a TI with name "" is executed.  I can't 100% reproduce but we are getting server crashing if that TI ends up in contention with other threads.  Either the other thread or this thread or both end up rolling back and our server crashes.  I was just wondering if anyone else has seen issues with unbound TI's and server crashing.

**To Reproduce**
Run script to clear data while activity on the server ends up in contention

**Expected behavior**
No server crashing

**Version**
TM1py 1.11.3
TM1SERVER_APP-AW64-ML-RTM-11.0.918.3-0

**Additional context**

**Comment by MariusWirtz** [comment]
I think we had spotted this behavior before when writing to an attribute cube with an unbound process.
I thought IBM had already addressed that.
@RobbyMeyers 
Can you confirm if the issue is occurring during attribute updates?

In any case, a Server crash needs to be raised with IBM, especially when your code is doing nothing wrong and the crash happens only under certain conditions.

We might be able to help you find a workaround, though. Please check if this resolves the issue:
- In your write operations, you could put a condition on the `use_blob` argument so that is uses `True` for all normal cubes and `False` for cubes that start with the prefix `}ElementAttributes_`

@sbethea-cw
Didn't you face a similar issue recently? How did you resolve it?

**Comment by RobbyMeyers** [comment]
The issue we are seeing now is when trying to clear the cube.  In this case we are clearing out an }ElementSecurity cube before writing to it.

Actually the behavior when using a cells.write actually seems to provide a name for the TI process when using a blob.
Process "tm1py.8aea51a60a75":  finished executing normally, elapsed time 3.39 seconds

Code

```
Clear }ElementSecurity cube data
try:
    clear_mdx = 'SELECT NON EMPTY {[}Groups].[}Groups].MEMBERS} ON 0, NON EMPTY {[' + dimension + '].[' + dimension + '].MEMBERS} ON 1 FROM [' + security_cube + ']'
    tm1_target.cells.clear_with_mdx(cube=security_cube, mdx=clear_mdx)
except TM1py.Exceptions.Exceptions.TM1pyRestException:
    print('Unable to clear data from ' + security_cube)
    sys.exit()
else:
    print('Cleared data from ' + security_cube)

```
Ill see if I can raise the issue with IBM however the issue is not 100% reproducible yet.  It seems to only be an issue if another thread ends up in contention with it.

I am currently testing a workaround of using the bedrock TI to clear the cube data.

**Comment by sbethea-cw** [comment]
@MariusWirtz  - yes, I faced this issue when writing to `}ElementAttributes_` cubes.  According to IBM, the issue was caused by multiple cube lock exceptions rolling back and causing the server to crash.

This issue was addressed in PA 2.0.9.16 (PH50104 - see fix list link below), but they also suggested changing the following tm1s.cfg parameter: `ReduceCubeLockingOnDimensionUpdate=T`

https://www.ibm.com/support/pages/system/files/inline-files/ibm_fixlist_planning_analytics_20916-2.pdf
https://www.ibm.com/docs/en/planning-analytics/2.0.0?topic=file-reducecubelockingondimensionupdate
